crates/lang/src/traits.rs

```diff
@@ -120,3 +121,63 @@ pub trait MessageMut: FnInput + FnOutput + FnSelector + FnState {
 /// Indicates that some compile time expression is expected to be `true`.
 #[doc(hidden)]
 pub trait True {}
+
+/// The ECDSA compressed public key.
+#[derive(Debug, Copy, Clone)]
+pub struct ECDSAPublicKey(pub [u8; 33]);
```

Comment:
Is there a reason why you've put this into `traits.rs`? From what I see there is no trait logic and my intuition would have been to put it into `env_access.rs`.